docs/dashboards/cloud-posture.asciidoc

```diff
@@ -1,2 +1,14 @@
 [[cloud-posture-dashboard]]
 = Cloud Posture Dashboard
+
+The Cloud Posture dashboard summarizes how your Kubernetes configuration measures up to security benchmarks. 
+
+NOTE: To learn how to collect this data, refer to <<kspm, Kubernetes security posture management>>.
+
+image::images/cloud-sec-dashboard.png[The Cloud Security dashboard]
+
+The first row of cards (Cloud Posture Score, Failed Findings, and Open Cases) summarizes your overall cloud security posture by aggregating data from all monitored Kubernetes clusters. Each subsequent row summarizes the posture of an individual Kubernetes cluster.
+
+The Cloud Posture Score card shows the percentage of your findings that passed over time. Hover your mouse to see precisely when the data was collected.
+
+The Failed Findings card shows failed findings grouped by CIS Section. Click any of the section names to view the failed findings from that section on the <<kspm, Findings page>>.
```

Comment:
I don't know if the audience will immediately know what a "CIS Section" is. Wouldn't hurt to at least spell it out, maybe even explain a bit more detail/context:
```suggestion
The Failed Findings card shows failed findings grouped by Center for Internet Security (CIS) benchmark categories. Click any section name to view its failed findings on the <<kspm, Findings page>>.
```